### PR TITLE
Replace LP oracle addresses

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

**Comment by gbalabasquer** [comment]
Please notice I'd commented the DAIUSDC one as it is a pair of `DSValue`. I think is pointless to be running this each time when there are no updates. The only thing to take into account is as soon as the spell is cast, it will need to be run manually the lp twice and then the spotter (this last one only if actually someone called it in the meanwhile there weren't prices).

**Comment by gbalabasquer** [comment]
@dizzy @grandizzy please confirm if you are deploying it or if somehow we should take care of it.

**Comment by grandizzy** [comment]
> @dizzy @grandizzy please confirm if you are deploying it or if somehow we should take care of it.

@gbalabasquer please deploy it on your side and let me know the address when done

**Review by grandizzy — APPROVED** [review]
LGTM, I double checked addresses to match new deployment

File: src/MegaPoker.sol [hunk 1]
```diff
@@ -69,7 +69,7 @@ contract MegaPoker is PokingAddresses {
         (ok,) = univ2daieth.call(abi.encodeWithSelector(0x18178358));
         (ok,) = univ2wbtceth.call(abi.encodeWithSelector(0x18178358));
         (ok,) = univ2usdceth.call(abi.encodeWithSelector(0x18178358));
-        (ok,) = univ2daiusdc.call(abi.encodeWithSelector(0x18178358));
+        // (ok,) = univ2daiusdc.call(abi.encodeWithSelector(0x18178358));
```
Comment (brianmcmichael) [inline src/MegaPoker.sol:72]: We should continue to poke this so that fees are accounted for so users can generate more Dai against their deposits.

Comment (gbalabasquer) [reply src/MegaPoker.sol]: Done

**Review by brianmcmichael — APPROVED** [review]

**Comment by talbaneth** [comment]
should we merge this PR to master?

**Pull request merged**